descriptors/ModuleDescriptor-template.json

```diff
@@ -86,13 +86,7 @@
             "orders.item.reopen"
           ],
           "modulePermissions": [
-            "modperms.orders.item.put",
-            "orders-storage.titles.item.get",
-            "orders-storage.configuration.prefixes.collection.get",
-            "orders-storage.configuration.suffixes.collection.get",
-            "finance.funds.item.get",
-            "finance.transactions.collection.get",
-            "finance.transactions.batch"
```

Comment:
These module permissions are used in source code when updating an order, why they are deleted?